schedule/functional/extra_tests_textmode.yaml

```diff
@@ -108,6 +108,7 @@ conditional_schedule:
                 - console/libjpeg_turbo
                 - console/valkey
                 - '{{velociraptor_tests}}'
+                - console/apache2_php_fpm
```

Comment:
Does it also need to add to sle?

Reply:
Neither SLE 15sp7 nor SLE16 is using this yaml schedule file